server/main.lua

```diff
@@ -16,12 +16,20 @@ local playerState
 
 AddEventHandler('QBCore:Server:OnPlayerLoaded', function()
 	playerState = Player(source).state
+	playerState:set(DEATH_STATE_STATE_BAG, Config.DeathState.ALIVE, true)
 	playerState:set(BLEED_LEVEL_STATE_BAG, 0, true)
 	for bodyPartKey in pairs(Config.BodyParts) do
 		playerState:set(BODY_PART_STATE_BAG_PREFIX .. bodyPartKey, nil, true)
 	end
 end)
 
+AddStateBagChangeHandler(DEATH_STATE_STATE_BAG, nil, function(bagName, _, value)
+    local playerId = string.sub(bagName, 8)
```

Comment:
You should use this native here: https://docs.fivem.net/natives/?_0xA56135E0

Reply:
done